src/metabase/models/card.clj

```diff
@@ -553,7 +553,9 @@
   (let [correct-collection-id (t2/select-one-fn :collection_id [:model/Dashboard :collection_id] (:dashboard_id card))
         invalid? (or (and (contains? card :collection_id)
                           (not= correct-collection-id (:collection_id card)))
-                     (not (contains? #{:question "question" nil} (:type card)))
+                     ;(not (contains? #{:question "question" nil} (:type card)))
+                     ;; TODO this might cause havoc somewhere?
+                     (not (contains? #{:question "question" :model "model" nil} (:type card)))
```

Comment:
I kicked the tires a bit by adding an editable to a dashboard, using it to edit stuff, and attaching filters, and for our purposes everything is fine.

This hackery will be fixed if we add a `:type` later.